.vscode/launch.json

```diff
@@ -0,0 +1,38 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "type": "node",
+            "request": "launch",
+            "name": "Launch Codegen",
+            "runtimeExecutable": "npm",
```

Comment:
Can we make this pnpm?

Reply:
Definitely, I'll do that in a separate PR since we'll need to add the pnpm engine to the package.json too.